src/service/packet_router.proto

```diff
@@ -6,7 +6,12 @@ import "region.proto";
 import "data_rate.proto";
 
 message packet_router_packet_report_v1 {
-  uint64 gateway_timestamp_ms = 1;
+  enum packet_type {
+    join = 0;
+    uplink = 1;
+  }
+
+  uint64 gateway_tmst = 1;
```

Comment:
Can we safely make this change or do we need to deprecate that old field 1 and add this as a new one?

Reply:
no.. it's just a name change.. same time.. and the values actually reflect the new name.. so no behavior change either